includes/class-wc-cardknox-api.php

```diff
@@ -29,7 +29,7 @@ class WC_Cardknox_API {
 	/**
 	 * Cardknox API Endpoint
 	 */
-	const ENDPOINT = 'https://x1.cardknox.com/gateway';
+	const ENDPOINT = 'https://x3.cardknox.com/gateway';
```

Comment:
Why is this needed?

Reply:
@mstein-ck , when I apple pay test, it is not working. After that, update the 'https://x3.cardknox.com/gateway' endpoint. It works. also, I checked on the demo server; Magento uses this endpoint.

Reply:
@pritesh-lr Do you have an example of a payment that failed?

Reply:
ApplePay is not working so I updated `https://x3.cardknox.com/gateway` endpoint. but credit card is working fine with `https://x1.cardknox.com/gateway`